types/lzma/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for lzma 2.3
+// Project: https://github.com/LZMA-JS/LZMA-JS
+// Definitions by: janKir <https://github.com/janKir>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ If this module has methods, declare them as functions like so.
+ */
+export function compress(
+    input: string | Uint8Array,
```

Comment:
https://github.com/LZMA-JS/LZMA-JS#how-to-use

> A Node.js Buffer or a Uint8Array instance counts as an array of bytes.

Does the definition fully cover supported types?